src/main/kotlin/uk/gov/dluhc/printapi/config/JacksonConfiguration.kt

```diff
@@ -21,6 +21,7 @@ class JacksonConfiguration {
             .addModule(KotlinModule.Builder().build())
             .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
             .disable(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS)
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
I thought this was already added as part of the development process for extending the ero response to include new contact fields so that the application won't fail. 

Since we are happy to configure the deserializer to not fail for unknown fields in other apis, I think it will ease our jobs, if we have it here as well.

Reply:
So did I... https://github.com/cabinetoffice/eip-ero-print-api/pull/93/files